virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
 
-xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[For secondary networks], use the `bridge` binding method.
+xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc#virt-attaching-vm-to-external-network[For secondary networks], use the `bridge` binding method.
```

Comment:
"secondary networks" should be "external networks" for the sake of consistency. If you are changing the title for the assembly, it makes sense to change references to it as well.

Reply:
Heartily agree... :-) I wondered about this but didn't want to make the call. There is a lot of use of the term "secondary" in assembly and module names, and obviously topics. I would recommend creating another Jira to address this inconsistency. But yes, I don't think calling something "secondary" is useful. All instances of "secondary" in this PR have been changed with the exception of existing assembly, module, and file names.